config/locales/de.yml

```diff
@@ -101,8 +100,49 @@ de:
 
   save: Speichern
   discard: Änderungen verwerfen
+
   dashboard:
     how_it_works: So funktioniert 100eyes…
+
+  setting:
+    setting:
+      one: Einstellung
+      other: Einstellungen
+    form:
+      groups:
+        project_settings: Projekt
+        telegram: Telegram
+        sendgrid: Sendgrid
+      project_name:
+        label: Name
+      onboarding_token:
+        label: Onboarding Token
+      telegram_bot_api_key:
+        label: API Key
+      telegram_bot_username:
+        label: Username
+      sendgrid_username:
+        label: Name
+      sendgrid_password:
+        label: API key
+      sendgrid_domain:
+        label: Domain
+      sendgrid_from:
+        label: Absender
+      inbound_email_password:
+        label: Inbound Passwort
+        help: Bei Sendgrid "%{callback_url}" unter "Inbound Parse" eintragen
```

Comment:
I think this could be a more helpful, friendlier help text, fort example:

> Trage diese URL in den Sendgrid-Einstellungen unter „Inbound Parse” ein. Ein detaillierte Anleitung findest du in der [Sendgrid-Dokumentation](https://sendgrid.com/docs/for-developers/parsing-email/setting-up-the-inbound-parse-webhook/).